plugins/catalog-backend-module-aws/src/processors/AwsOrganizationCloudAccountProcessor.ts

```diff
@@ -98,23 +106,27 @@ export class AwsOrganizationCloudAccountProcessor implements CatalogProcessor {
 
     this.logger?.info('Discovering AWS Organization Account objects');
 
-    (await this.getAwsAccounts())
-      .map(account => this.mapAccountToComponent(account))
-      .filter(entity => {
-        if (location.target !== '') {
-          if (entity.metadata.annotations) {
-            return (
-              entity.metadata.annotations[ORGANIZATION_ANNOTATION] ===
-              location.target
-            );
+    try {
+      (await this.getAwsAccounts())
+        .map(account => this.mapAccountToComponent(account))
+        .filter(entity => {
+          if (location.target !== '') {
+            if (entity.metadata.annotations) {
+              return (
+                entity.metadata.annotations[ORGANIZATION_ANNOTATION] ===
+                location.target
+              );
+            }
+            return false;
           }
-          return false;
-        }
-        return true;
-      })
-      .forEach(entity => {
-        emit(processingResult.entity(location, entity));
-      });
+          return true;
+        })
+        .forEach(entity => {
+          emit(processingResult.entity(location, entity));
+        });
+    } catch (e) {
+      this.logger?.error(e);
```

Comment:
don't we need to emit the error at this point? cc @freben

Reply:
Yeah. In fact it looks like temporary test code right? Might as well just remove the try/catch and let it bubble out to the caller instead.

Reply:
The reason we included this was we were actually having problems with the code throwing errors and we didn't see any output in the logs. I'll revert this code since it isn't directly related to the PR.